tests/NRedisStack.Tests/Search/IndexCreationTests.cs

```diff
@@ -0,0 +1,126 @@
+    public void TestMissingFields()
+    {
+        IDatabase db = redisFixture.Redis.GetDatabase();
+        db.Execute("FLUSHALL");
+        var ft = db.FT(2);
+        var vectorAttrs = new Dictionary<string, object>()
+        {
+            ["TYPE"] = "FLOAT32",
+            ["DIM"] = "2",
+            ["DISTANCE_METRIC"] = "L2",
+        };
+        Schema sc = new Schema()
+           .AddTextField("text1", 1.0, missingIndex: true)
+           .AddTagField("tag1", missingIndex: true)
+           .AddNumericField("numeric1", missingIndex: true)
+           .AddGeoField("geo1", missingIndex: true)
+           .AddGeoShapeField("geoshape1", Schema.GeoShapeField.CoordinateSystem.FLAT, missingIndex: true)
+           .AddVectorField("vector1", Schema.VectorField.VectorAlgo.FLAT, vectorAttrs, missingIndex: true);
+
+        var ftCreateParams = FTCreateParams.CreateParams();
+        Assert.True(ft.Create(index, ftCreateParams, sc));
+
+        var hashWithMissingFields = new HashEntry[] { new("field1", "value1"), new("field2", "value2") };
+        db.HashSet("hashWithMissingFields", hashWithMissingFields);
+
+        Polygon polygon = new GeometryFactory().CreatePolygon(new Coordinate[] { new Coordinate(1, 1), new Coordinate(10, 10), new Coordinate(100, 100), new Coordinate(1, 1), });
+
+        var hashWithAllFields = new HashEntry[] { new("text1", "value1"), new("tag1", "value2"), new("numeric1", "3.141"), new("geo1", "-0.441,51.458"), new("geoshape1", polygon.ToString()), new("vector1", "aaaaaaaa") };
+        db.HashSet("hashWithAllFields", hashWithAllFields);
+
+        var result = ft.Search(index, new Query("ismissing(@text1)"));
+        Assert.Equal(1, result.TotalResults);
+        Assert.Equal("hashWithMissingFields", result.Documents[0].Id);
+
+        result = ft.Search(index, new Query("ismissing(@tag1)"));
+        Assert.Equal(1, result.TotalResults);
+        Assert.Equal("hashWithMissingFields", result.Documents[0].Id);
+
+        result = ft.Search(index, new Query("ismissing(@numeric1)"));
+        Assert.Equal(1, result.TotalResults);
+        Assert.Equal("hashWithMissingFields", result.Documents[0].Id);
+
+        result = ft.Search(index, new Query("ismissing(@geo1)"));
+        Assert.Equal(1, result.TotalResults);
+        Assert.Equal("hashWithMissingFields", result.Documents[0].Id);
+
+        result = ft.Search(index, new Query("ismissing(@geoshape1)"));
+        Assert.Equal(1, result.TotalResults);
+        Assert.Equal("hashWithMissingFields", result.Documents[0].Id);
+
+        result = ft.Search(index, new Query("ismissing(@vector1)"));
+        Assert.Equal(1, result.TotalResults);
+        Assert.Equal("hashWithMissingFields", result.Documents[0].Id);
+    }
+
+    [SkipIfRedis(Is.OSSCluster, Comparison.LessThan, "7.3.240")]
+    public void TestEmptyFields()
+    {
+        IDatabase db = redisFixture.Redis.GetDatabase();
+        db.Execute("FLUSHALL");
```

Comment:
I guess because there's no distinguish before integration and unit tests, you have to run it in each integration test?

Maybe as a tech debt it makes sense to create a distinguish between tests, and make sure that FLUSHALL runs before each integration test automatically. I know that usually framework provide some kind of "hook" for it, methods like `onSetUp()` or smth simillar